src/NServiceBus.Metrics.PerformanceCounters.sln.DotSettings

```diff
@@ -557,6 +557,9 @@ II.2.12 &lt;HandlesEvent /&gt;&#xD;
 	<s:String x:Key="/Default/CodeStyle/Naming/XamlNaming/UserRules/=XAML_005FFIELD/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/XamlNaming/UserRules/=XAML_005FRESOURCE/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpFileLayoutPatternsUpgrade/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpKeepExistingMigration/@EntryIndexedValue">True</s:Boolean>
```

Comment:
What are these changes?

Reply:
Resharper :wink:

Reply:
Right, but are they Resharper changes that we want? What's the diff of this file compared to the RepoStandards version?

Reply:
These are not really a RepoStandards kind of thing. They mean that ReSharper was updated and so a migrator was applied so you don't have to do it again. I can revert it, but when the next person with R# opens the solution, it will come right back.

https://resharper-support.jetbrains.com/hc/en-us/community/posts/206653075--sln-DotSettings-file-keeps-being-changed

Reply:
> These are not really a RepoStandards kind of thing. 

https://github.com/Particular/RepoStandards/blob/master/src/RepoName.sln.DotSettings would disagree with that.

Does that mean we need to update the RepoStandards file in some way?

Reply:
No, then any project that took RepoStandards would skip any of the migrators, which _could_ be OK, but they're a black box so 🤷‍♂. Honestly I wish R# would separate definitions of inspections (very appropriate for RepoStandards) and this ephemeral stuff into separate files.

Reply:
I just wish R# could do what it needs without needing artifacts checked into the repo at all. Oh well.